dm/ui/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: ['@ti-fe/eslint-config'],
```

Comment:
just a question: is this file is the lint rule for JS project ? why all rule is empty?

Reply:
Yes, it extends from `@ti-fe/eslint-config`, which is a shared collection of rules used by our front-enders, you can also specify other rules here in this file or override it.